src/components/compound/DlCharts/charts/DlDoughnutChart/DlDoughnutChart.vue

```diff
@@ -312,6 +330,9 @@ export default defineComponent({
 
         /** Methods */
         const hideData = (item: { index: number; hidden: boolean }) => {
+            if (attrs.onItemClick as ItemClickHandler) {
+                return (attrs.onItemClick as ItemClickHandler)(item.index)
```

Comment:
Why should we ignore the rest of the function ?

Reply:
Well because the rest of the function hides the clicked item - and here we are providing the custom click handler

Reply:
(in the context of DAT-64643 that would be setting dataset filter)

Reply:
The original requirement was to click on the chart itself, not on the legend.
Clicking on the legend should really hide the section in the chart

Reply:
okay, removed it.